edkrepo/common/ui_functions.py

```diff
@@ -47,6 +51,39 @@ def display_git_output(output_data, verbose=False):
     if verbose and output_data[2]:
         print(output_data[2])
 
+def print_info_msg(info_msg, header=True):
+    """
+    Displays informational message with (default) or without header. 
+    """
+    init_color_console("init")
+    if header:
+        info_msg_formatted = "{}{}Info: {}{}{}".format(Style.BRIGHT, Fore.CYAN, Style.RESET_ALL, Fore.CYAN, info_msg)
+    else:
+        info_msg_formatted = "{}{}".format(Fore.CYAN, info_msg)
+    print(info_msg_formatted)
+
+def print_warning_msg(warning_msg, header=True):
+    """
+    Displays warning message with (default) or without header.
+    """
+    init_color_console("init")

```

Comment:
The parameter for this function is force_color_output and is a binary. Why are we passing in the string 'init' this will not work as intended. Note that the use of this function takes in the color argument you can see this in use in the log command. 
We must  update this function to take in a parameter for forcing color output and pass it through to the init_color_console command. We can  assign this parameter a default value so that if not passed it is None.